packages/@expo/config-plugins/src/utils/generateCode.ts

```diff
@@ -10,8 +10,8 @@ function getGeneratedSectionIndexes(
   tag: string
 ): { contents: string[]; start: number; end: number } {
   const contents = src.split('\n');
-  const start = contents.findIndex((line) => line.includes(`@generated begin ${tag}`));
-  const end = contents.findIndex((line) => line.includes(`@generated end ${tag}`));
+  const start = contents.findIndex((line) => new RegExp(`@generated begin ${tag} -`).test(line));
```

Comment:
I'm a bit hesitant by adding ` -` to this regex, mostly by [this comment](https://github.com/expo/expo/blob/b029de00ba7006db5bfc5f6ce9091229ef0e859c/packages/%40expo/config-plugins/src/utils/generateCode.ts#L119C37-L119C37):
```
// Everything after the `${tag} ` is unversioned and can be freely modified without breaking changes.
return `${comment} @generated begin ${tag} - expo prebuild (DO NOT MODIFY) ${hashKey}`;
```

We could still go with this. But, if that concern is shared, we could also force the `tags` to be slugs to replace spaces with `-`. With that, we can just test if there is no space at the end, but not sure about the current usage (are there a lot of tags without spaces currently being used).

Reply:
I see evan already commented, closing this.